### PR TITLE
Improve the discovery of the command line options

### PR DESCRIPTION
**Pull request by DelazJ**
by setting them as subsection titles instead of simple paragraph whose rendering is the same as their text description, hence not easy to find. Incidentally, this also allows to directly link the option when referring to it in a discussion.
Limit the TOC depth at 2 to avoid the tens of options shown in the list.

However I don't quite know what would be the best meaningful display for the title, with or without \``, -- or option
Opinion?
image

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

**Comment by havatv** [comment]
The `--snapshot` variant looks fine to me.  No need to add ` option`, and useful to have the two leading dashes.

**Comment by Gustry** [comment]
I don't see the snapshot one in the screenshot. But I prefer either project or extent.

**Comment by DelazJ** [comment]
@Gustry I did not show snapshot because its description is rather long and it wouldn't help to show the 5 variants of title. In the pr, it should look like "lang"

**Comment by DelazJ** [comment]
Here we go with `--project``

**Comment by DelazJ** [comment]
I wonder if we can review the order and associate features. eg, move snapshot and take-screenshots at the end, move profile-related options next to customization related ones... i don't know

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 1]
```diff
@@ -1407,19 +1409,22 @@ be added after ``--snapshot``. For example::
...
-``--project`` option
+``--project``
+..............
 
 Starting QGIS with an existing project file is also possible. Just add the
 command line option ``--project`` followed by your project name and QGIS will
 open with all layers in the given file loaded.
 
-``--extent`` option
+``extent``
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
``--extent``
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 2]
```diff
@@ -1394,7 +1395,8 @@ returns::
         set to load on startup using the following command:
         ``qgis ./raster/landcover.img ./gml/lakes.gml``
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: ``--version``
.............

Returns the QGIS version.

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 3]
```diff
@@ -1407,19 +1409,22 @@ be added after ``--snapshot``. For example::
 
   qgis --snapshot my_image.png --width 1000 --height 600 --project my_project.qgs
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]: ``--width``
...........

Width of the snapshot to be emitted (used with ``--snapshot``).

``--height``
............

Height of the snapshot to be emitted (used with ``--snapshot``).

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 4]
```diff
@@ -1407,19 +1414,32 @@ be added after ``--snapshot``. For example::
...
+...........
+
+This option returns the width of the snapshot to be emitted (used with ``--snapshot``).
+
+``--height``
+............
+
+This option returns the height of the snapshot to be emitted (used with ``--snapshot``).
+
+`--lang``
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
``--lang``
```

File: source/docs/user_manual/introduction/qgis_configuration.rst [hunk 5]
```diff
@@ -1394,7 +1395,13 @@ returns::
         set to load on startup using the following command:
         ``qgis ./raster/landcover.img ./gml/lakes.gml``
 
-``--snapshot`` option
+``--version``
+..............
+
+This option returns QGIS version.
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_configuration.rst]:
```suggestion
This option returns QGIS version information.
```

**Pull request merged**